src/Providers/GeocodeService.js

```diff
@@ -5,8 +5,34 @@ EsriLeafletGeocoding.Controls.Geosearch.Providers.GeocodeService = EsriLeafletGe
   },
 
   suggestions: function(text, bounds, callback){
-    callback(undefined, []);
-    return false;
+
+    if (this.options.supportsSuggest) {
```

Comment:
i'm using the result of `_confirmSuggestSupport()` it to avoid asking for suggestions from services that don't support the operation here.

are you saying to blow away the conditional logic entirely?